Makefile

```diff
@@ -129,10 +137,62 @@ lint/makefile: checkmake
 	@echo "Linting Makefile..."
 	checkmake --config .checkmake Makefile
 
-lint/license: ## Check license headers
-lint/license: go-license
+# License header targets
+
+.PHONY: license-header-check
+license-header-check: ## Check license headers in source files
 	@echo "Checking license headers..."
-	go-license --config=license.yml --verify .
+	@licRes=$$(for pattern in $(LICENSE_FILE_PATTERNS); do \
+	    find . -type f -iname $$pattern ! \( $(LICENSE_EXCLUDE_PATHS) \); \
+	done | while read f; do \
+	    if ! grep -q "Copyright The OpenTelemetry Authors" "$$f" || \
+	       ! grep -q "SPDX-License-Identifier: Apache-2.0" "$$f"; then \
+	        echo "$$f"; \
+	    fi; \
+	done); \
+	if [ -n "$${licRes}" ]; then \
+	    echo "Missing license header in the following files:"; echo "$${licRes}"; \
+	    exit 1; \
+	fi
+	@echo "All files have proper license headers"
+
+.PHONY: license-header-fix
+license-header-fix: ## Add missing license headers to source files
```

Comment:
lint/license-header/fix: ## Add missing license headers to source file

Reply:
done